release.py

```diff
@@ -0,0 +1,490 @@
+import base64
+import getpass
+import git
+import json
+import os
+import re
+import subprocess
+import urllib2
+
+try:
+    from setuptools import Command
+except ImportError:
+    from distutils.core import Command
+
+REPOSITORY_API = 'https://api.github.com/repos/prestodb/presto-admin'
+CURRENT_DIRECTORY = os.path.dirname(os.path.realpath(__file__))
+RELEASE_LIST_DOC_PATH = os.path.join(CURRENT_DIRECTORY, 'docs/release.rst')
+RELEASE_DOCS_DIRECTORY = os.path.join(CURRENT_DIRECTORY, 'docs/release/')
+DIST_DIRECTORY = os.path.join(CURRENT_DIRECTORY, 'dist')
+
+
+class release(Command):
```

Comment:
Most of the methods in here are static. Maybe it makes sense to break out github and pypi files that handle the inner workings of pushing to github and pypi and invoke that functionality here? That might also have the benefit of helping to make clear what the public interfaces are and what functions are internal implementation details.